com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingBSDF.hlsl

```diff
@@ -607,6 +607,16 @@ bool RandomWalk(float3 position, float3 normal, float3 diffuseColor, float3 mean
         result.exitNormal = intersection.value;
     }
 
+// if the material is transmissive, pick a side for the normal randomly to properly handle thin surfaces (case 1329403)
+#ifdef _MATERIAL_FEATURE_TRANSMISSION
```

Comment:
I will accept the PR for the fix, but let's have in mind that _MATERIAL_FEATURE_TRANSMISSION isn't a standard name for "transmission". This is define only per Material and Material don't have obligation to reuse this name (on raster side). We may need to revisit how we handle transmission from a general point of view later.